src/app/vault/ciphers.component.html

```diff
@@ -34,6 +34,11 @@
                                 <i class="fa fa-fw fa-clipboard"></i>
                                 {{'copyPassword' | i18n}}
                             </a>
+                            <a class="dropdown-item" href="#" appStopClick
+                                *ngIf="c.type === cipherType.Login && c.login.canLaunch" (click)="launch(c.login)">
```

Comment:
You appear to be passing a `LoginView` object here, yet your `lunch` function is accepting a `LoginUriView`.

Reply:
Oops! Should be fixed now. Sorry, this is my first foray into TypeScript and Angular...